mutation-testing/shell-scripts/remove-deleted-file-lines.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+input_file="git.diff"
+temp_file="temp_diff_file.diff"
+
+# Reverse the file, remove 4 lines after '+++ /dev/null', then reverse it back (editors can't go backwards - to remove lines above)
+tac "$input_file" > "$temp_file" && mv "$temp_file" "$input_file"
```

Comment:
I'd feel better here if there was a line checking if the binary is on the host, and if it's in `$PATH`. ex:
```bash
for cmd in tac awk sed; do
	command -v "${cmd}" > /dev/null 2>&1 || echo "Missing command: ${cmd}"
done
```